rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -32,6 +31,7 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
     }
 
     packerCommand << absoluteTemplateFilePath
+    packerCommand.removeAll([null])
```

Comment:
@duftler I did this to handle the null in the baseCommand and also for the rest of the packerCommand.